src/puppeteer.js

```diff
@@ -150,6 +94,66 @@ const getPuppeteerOrThrow = (puppeteerModule = 'puppeteer') => {
     }
 };
 
+// TODO yin: `@property ...` wont work. It's may cause undefined property error in Typescript.
+//  It would be great if we can extend `LaunchOptions` from `puppeteer`. There is a GitHub issue for that:
+//  https://github.com/Microsoft/TypeScript/issues/20077
```

Comment:
Won't do. 

```diff
diff --git src/puppeteer.js src/puppeteer.js
index c44d95e..3d36478 100644
--- src/puppeteer.js
+++ src/puppeteer.js
@@ -98,17 +98,16 @@ const getPuppeteerOrThrow = (puppeteerModule = 'puppeteer') => {
 //  It would be great if we can extend `LaunchOptions` from `puppeteer`. There is a GitHub issue for that:
 //  https://github.com/Microsoft/TypeScript/issues/20077
 /**
- * Apify extends the launch options of Puppeteer.
- * You can use any of the
+ * Apify extends the launch options of Puppeteer. In addition to configuration options defined by
+ * {@link LaunchPuppeteerOptionsExtension} you can use any of the
  * <a href="https://pptr.dev/#?product=Puppeteer&show=api-puppeteerlaunchoptions" target="_blank"><code>puppeteer.launch()</code></a>
  * options in the [`Apify.launchPuppeteer()`](../api/apify#module_Apify.launchPuppeteer)
- * function and in addition, all the options available below.
- *
- * @typedef {Object} LaunchPuppeteerOptions
- * @property ...
- *   You can use any of the
- *   <a href="https://pptr.dev/#?product=Puppeteer&show=api-puppeteerlaunchoptions" target="_blank"><code>puppeteer.launch()</code></a>
- *   options.
+ * function.
+ * @typedef {LaunchOptions & LaunchPuppeteerOptionsExtension} LaunchPuppeteerOptions
+ */
+/**
+ * Extended launch options for Puppeteer. See {@link LaunchPuppeteerOptions} for more details.
+ * @typedef {Object} LaunchPuppeteerOptionsExtension
  * @property {String} [proxyUrl]
  *   URL to a HTTP proxy server. It must define the port number,
  *   and it may also contain proxy username and password.
```

JSDoc gets confused by this:

```
Invalid type expression "LaunchOptions & LaunchPuppeteerOptionsExtension": Expected "|" but "&" found.
```